src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/Example3.java

```diff
@@ -16,10 +16,9 @@ class Example3 {
 
   public void method0() {
     int mode = 0;
+    int x;
     switch (mode) {
-      case 1:
-        int x = 1;
-        break;
+      case 1: {x = 1;}   // violation '}' at column 22 should be alone on a line'
```

Comment:
Keep original case. Make this one `int y = 1;`

Reply:
done